test/tests/functional/pbs_qsub_script.py

```diff
@@ -81,3 +81,31 @@ def test_qsub_basic_job(self):
             job_output_file = job_status[0]['Output_Path'].split(':')[1]
         rc = self.du.cmp(fileA=fn, fileB=job_output_file, runas=TEST_USER)
         self.assertEqual(rc, 0, 'cmp of job files failed')
+
+    def test_qsub_line_extensions(self):
+        """
+        This test case ensures that only #PBS directive lines are treated
+        as candidates for line extension
+        """
+        script = """#!/bin/sh
+        #PBS -m n
+        # This is a test comment that shouldn't be extended \\
+        cat $0
+        """
```

Comment:
Please extend a #PBS directive as well as a comment.

Reply:
Updated script includes extended directive, with a blank extended line as well to verify this works correctly